ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-stable-openshift-logging-6.3.yaml

```diff
@@ -63,6 +63,38 @@ tests:
     - ref: install-operators
     - chain: openshift-e2e-test-logging-qe
     workflow: cucushift-installer-rehearse-azure-ipi
+- as: azure-ipi-amd-f999-logging-custom-tls-profile
+  capabilities:
+  - intranet
+  cron: 8 8 29 2 *
+  steps:
+    cluster_profile: azure-observability
+    env:
+      BASE_DOMAIN: observability.azure.devcluster.openshift.com
+      FORCE_SUCCESS_EXIT: "true"
+      LOGGING_INDEX_IMAGE: brew.registry.redhat.io/rh-osbs/iib:log6.3
+      LOGGING_TLS_PROFILE_CONFIG: '{"type": "Custom", "custom": { "ciphers": ["ECDHE-ECDSA-CHACHA20-POLY1305","ECDHE-RSA-CHACHA20-POLY1305","ECDHE-RSA-AES128-GCM-SHA256","ECDHE-ECDSA-AES128-GCM-SHA256"],"minTLSVersion":
+        "VersionTLS12"}}'
+      OCP_ARCH: amd64
+      OPERATORS: |
+        [
+            {"name": "loki-operator", "source": "qe-app-registry", "channel": "stable-6.3", "install_namespace": "openshift-operators-redhat","target_namespaces": "", "operator_group": "openshift-operators-redhat"},
+            {"name": "cluster-logging", "source": "qe-app-registry", "channel": "stable-6.3", "install_namespace": "openshift-logging","target_namespaces": "", "operator_group": "openshift-logging"},
+            {"name": "cluster-observability-operator", "source": "redhat-operators", "channel": "stable", "install_namespace": "openshift-cluster-observability-operator","target_namespaces": "", "operator_group": "openshift-cluster-observability-operator"},
+            {"name": "amq-streams", "source": "redhat-operators", "channel": "stable", "install_namespace": "openshift-operators","target_namespaces": "", "operator_group": ""}
+        ]
+      TEST_ADDITIONAL: ""
+      TEST_FILTERS: Disruptive
+      TEST_IMPORTANCE: High
+      TEST_PARALLEL: "1"
+      TEST_SCENARIOS: sig-openshift-logging
+      TEST_TIMEOUT: "30"
+    test:
+    - ref: openshift-logging-patch-tlssecurityprofile-apiserver
+    - ref: enable-qe-catalogsource-logging
```

Comment:
If you want to set the catalog source index when triggering the job, you can use step `openshift-logging-install-konflux-catalogsource`, and no need to set `LOGGING_INDEX_IMAGE`.

Reply:
Updated